src/brogue/Time.c

```diff
@@ -1333,6 +1333,8 @@ void monstersFall() {
     for (monst = monsters->nextCreature; monst != NULL; monst = nextCreature) {
         nextCreature = monst->nextCreature;
         if ((monst->bookkeepingFlags & MB_IS_FALLING) || monsterShouldFall(monst)) {
+            if (rogue.patchVersion >= 3) monst->bookkeepingFlags |= MB_IS_FALLING;
```

Comment:
I wonder if this is necessary, as the `||` in the above is between one case where the flag is set, and one where it isn't - it would be strange if further code implicitly relied on that - but def better to be safe!

Reply:
Yeah lol, the code in monstersFall seems to imply that MB_IS_FALLING may or may not be properly set before the function is called, but I'd rather set the flag whenever applicable because it seems like it's properly set every time it's supposed to change.

Reply:
Yes, I meant to say specifically the guarding by rogue.patchVersion, but no harm in it being there